src/main/java/com/ldtteam/domumornamentum/block/decorative/PillarBlock.java

```diff
@@ -0,0 +1,340 @@
+import org.jetbrains.annotations.Nullable;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public class PillarBlock extends AbstractBlock<PillarBlock> implements IMateriallyTexturedBlock, ICachedItemGroupBlock, EntityBlock
+{
+    public static final List<IMateriallyTexturedBlockComponent> COMPONENTS = ImmutableList.<IMateriallyTexturedBlockComponent>builder()
+        .add(new SimpleRetexturableComponent(new ResourceLocation("block/oak_planks"), ModTags.PILLAR_MATERIALS, Blocks.OAK_PLANKS))
+        .build();
+
+    private final List<ItemStack> fillItemGroupCache = Lists.newArrayList();
+    private static final Optional<VoxelShape> pillar_capital_shape = Stream.of(
+     Shapes.box(0, 0.875, 0, 1, 1, 1),
+     Shapes.box(0.0625, 0.8125, 0.0625, 0.9375, 0.875, 0.9375),
+     Shapes.box(0.125, 0.75, 0.125, 0.875, 0.8125, 0.875),
+     Shapes.box(0.4375, 0, 0.1875, 0.5625, 0.75, 0.8125),
+     Shapes.box(0.4375, 0, 0.1875, 0.5625, 0.75, 0.8125),
+     Shapes.box(0.4375, 0, 0.1875, 0.5625, 0.75, 0.8125),
+     Shapes.box(0.1875, 0, 0.4375, 0.8125, 0.75, 0.5625),
+     Shapes.box(0.1875, 0, 0.4375, 0.8125, 0.75, 0.5625),
+     Shapes.box(0.1875, 0, 0.4375, 0.8125, 0.75, 0.5625),
+     Shapes.box(0.1875, 0, 0.4375, 0.8125, 0.75, 0.5625),
+     Shapes.box(0.4375, 0, 0.1875, 0.5625, 0.75, 0.8125)
+     ).reduce((v1,v2) ->Shapes.join(v1, v2,BooleanOp.OR));
+
+    /**
+     * This block's name.
+     */
+    public static final String                      BLOCK_NAME     = "blockpillar";
+
+    /**
+     * The hardness this block has.
+     */
+    private static final float                      BLOCK_HARDNESS = 3F;
+
+    /**
+     * The resistance this block has.
+     */
+    private static final float                      RESISTANCE     = 1F;
+    private static final Property column = EnumProperty.create("column", PillarShapeType.class);
+
+    public PillarBlock()
+    {
+        super(BlockBehaviour.Properties.of(Material.GLASS).strength(BLOCK_HARDNESS, RESISTANCE));
+        this.registerDefaultState(this.stateDefinition.any().setValue(column,PillarShapeType.full_pillar));
+        setRegistryName(BLOCK_NAME);
+    }
+
+    /**
+     * Registry block at game registry.
+     *
+     * @param registry the registry to use.
+     */
+    @Override
+    public void registerItemBlock(final IForgeRegistry<Item> registry, final Item.Properties properties)
+    {
+        registry.register((new PillarBlockItem(this, properties)).setRegistryName(Objects.requireNonNull(this.getRegistryName())));
+    }
+
+    @Override
+    public VoxelShape getOcclusionShape(BlockState p_60578_, BlockGetter p_60579_, BlockPos p_60580_)
+    {
+        return pillar_capital_shape.orElse(Shapes.block());
+    }
+
+    public VoxelShape getVisualShape(BlockState p_53311_, BlockGetter p_53312_, BlockPos p_53313_, CollisionContext p_53314_) {
```

Comment:
make methods consistent 

{

}

bracing, same for if/else. Also always add @Override consistently and name the params